docs/_posts/examples/v1.0.0/0100-01-01-choropleth-joined-data-multiple-variables.html

```diff
@@ -73,18 +73,18 @@
 
 // Create a layer of state features, and when it's done
 // loading, run loadData
-var usLayer = L.mapbox.featureLayer()
-    .loadURL('{{site.baseurl}}/assets/data/us.geojson')
-    .addTo(map)
-    .on('ready', loadData);
+var usLayer = L.mapbox.featureLayer('{{site.baseurl}}/assets/data/us.geojson')
+    .on('ready', function () {
+        loadData(this.getGeoJSON())
+    }).addTo(map);
 
 // Grab the spreadsheet of data as JSON. If you have CSV
 // data, you should convert it to JSON with
 // http://shancarter.github.io/mr-data-converter/
-function loadData() {
+function loadData(geojson) {
     $.getJSON('{{site.baseurl}}/assets/data/censusdata.json')
-        .done(function(data) {
-            joinData(data, usLayer);
+        .done(function (data) {
+            joinData(data, geojson);
         });
 }
 
```

Comment:
@malwoodsantoro Two small fixes :)

We'll also need to update `joinData()` function, since we are passing the `usLayer`'s geojson to the function and no longer need to run `usLayer.getGeoJSON()`.

Looking at line 91, `function joinData(data, layer) {` the second parameter `layer` is no longer used, so we could rename it `usGeoJSON`:

```js
function joinData(data, usGeoJSON) {
```

Since the value of `usGeoJSON` is set as the function's parameter, we can replace lines 93-94 with:

```js
var byState = {};
```

Reply:
Ah, thanks @katydecorah 👍